package.json

```diff
@@ -24,28 +29,26 @@
     "passport-github": "^0.1.5",
     "pg": "^4.4.0",
     "rc": "~0.5.0",
+    "react": "^15.1.0",
+    "react-dom": "^15.1.0",
+    "react-router": "^2.4.1",
     "request": "^2.61.0",
     "sequelize": "^3.30.0",
     "underscore": "^1.8.3",
     "urlsafe-base64": "^1.0.0",
+    "webpack": "^2.6.1",
     "whatwg-fetch": "^1.0.0",
     "winston": "^2.3.1",
     "yamljs": "^0.2.8"
   },
   "scripts": {
     "postinstall": "npm run uswds && npm run migrate:up",
-    "build": "npm run browserify && npm run sass && npm run autoprefixer",
+    "build": "webpack -p",
     "start": "node index.js",
     "debug": "node debug index.js",
-    "browserify": "browserify frontend/main.js -o public/js/bundle.js -t [ stringify --extensions [ .html ] ] -t [ babelify --presets [ es2015 react stage-2 ] ]",
-    "sass": "node-sass frontend/sass/styles.scss -o public/styles",
-    "autoprefixer": "postcss --use autoprefixer -r public/styles/styles.css",
     "watch": "npm run uswds && node index.js & npm run watch:client",
     "watch:server": "nodemon index.js --ignore frontend/ --ignore .tmp/ --ignore node_modules/",
-    "watch:client": "npm run watch:client:sass & npm run watch:client:autoprefix & npm run watch:client:js",
-    "watch:client:sass": "node-sass -w -r frontend/sass/ -o public/styles",
-    "watch:client:autoprefix": "npm run sass && postcss --use autoprefixer -r -w public/styles/styles.css public/styles/styles.css",
-    "watch:client:js": "watchify frontend/main.js -o public/js/bundle.js -t [ stringify --extensions [.html] ] -t [ babelify --presets [ es2015 react stage-2 ] ]",
+    "watch:client": "webpack --progress --colors --watch",
```

Comment:
this task runs webpack in non-production mode, and rebuilds assets on file changes